tools/docs/manager.ts

```diff
@@ -70,6 +75,9 @@ sidebar_label: ${displayName}
         .join(', ');
       md += `This manager supports extracting the following datasources: ${escapedDatasources}.\n\n`;
 
+      md += '## References';
+      md += formatUrls(urls).replace('**References**:', '');
```

Comment:
```suggestion
      md += formatUrls(urls);
```

Instead of removing it here again, we should add a parameter which controls the header. 
Like a string which defaults to `**References**`

Reply:
sure, will do in separate PR